hera_qm/tests/test_ant_metrics.py

```diff
@@ -26,15 +26,10 @@ def test_calc_corr_stats():
                 (0, 1, 'nn'): np.array([[False, False], [False, False]])}
 
     # test normal operation
-    corr_stats = ant_metrics.calc_corr_stats(data_sum,data_diff,flags)
+    corr_stats = ant_metrics.calc_corr_stats(data_sum,data_diff)
     assert corr_stats[(0,1,'ee')] == 1
     assert corr_stats[(0,1,'nn')] == pytest.approx(0.9578,abs=1e-2)
 
-    # test with flags
-    flags[(0,1,'nn')][1,0] = True
-    corr_stats = ant_metrics.calc_corr_stats(data_sum,data_diff,flags)
-    assert corr_stats[(0,1,'nn')] == pytest.approx(0.9457,abs=1e-2)
-
     # test no diff data
```

Comment:
We may want to add a test for handling data that are zeros - we probably should have already had a test for this, but I'm particularly concerned now that flag handling has been removed.

Reply:
It seems that we broken the special identification of "totally dead" antennas (i.e. median 0 abs(vis)). I've added a new unit test to expose the issue and then I fixed it by undoing one of @AaronParsons's changes